developers/weaviate/configuration/indexes.md

```diff
@@ -101,6 +101,19 @@ If you're looking for a starting point for values, we would advise an `efConstru
 
 Note that the vector index type only specifies how the vectors of data objects are *indexed* and this is used for data retrieval and similarity search. How the data vectors are determined (which numbers the vectors contain) is specified by the `"vectorizer"` parameter which points to a [module](/developers/weaviate/modules/index.md) such as `"text2vec-contextionary"` (or to `"none"` if you want to import your own vectors). Learn more about all parameters in the data schema [here](/developers/weaviate/configuration/schema-configuration.md).
 
+### Asynchronous indexing
+
+:::info Available from version `v1.22`
+:::
+
+You can switch on the asynchronous indexing feature in Weaviate version `1.22` and higher. This feature decouples the object creation process in Weaviate from updating the vector index. This allows objects to be created faster, while the vector index is being updated in the background. This is especially useful for importing large amounts of data.
+
+This can be done by setting the `WEAVIATE_ASYNC_INDEXING` environment variable to `true` in the Weaviate configuration (e.g. in the `docker-compose.yml` file). This will enable asynchronous indexing for all classes.
+
+Please note that the un-indexed data will be searched by brute force (i.e. without the vector index) until the vector index has been updated. This means that the search performance will be slower until the vector index has been fully updated.
```

Comment:
Please mention the 100k limit for brute forcing. If the queue is larger than 100k the results will not be able to be searched straight away (from PR):

> For search queries, we merge the results of HNSW with the results of Brute force on the pending non-indexed vectors.
We limit the number of vectors passed to the Brute force algorithm (BruteForceSearchLimit parameter, defaul: 100k), to allow keeping a large number of vectors in memory without sacrificing too much query speed.
Once all vectors are indexed, we are expecting similar performance as before.